examples/simpleble/cpp/scan/scan.cpp

```diff
@@ -40,6 +40,8 @@ int main() {
 
         std::cout << "[" << i << "] " << peripheral_string << " " << connectable_string << std::endl;
 
+        std::cout << "    Tx Power: " << std::dec << peripherals[i].tx_power() << " dBm" << std::endl;
```

Comment:
You can add `address_type` as well without fear of blowing up if you apply my suggestion on this below.